thorlcr/msort/tsortm.cpp

```diff
@@ -441,10 +438,9 @@ class CSortMaster: public IThorSorterMaster, public CSimpleInterface
             mpp++;
         }
         if (keyserializer&&cosort&&!partitioninfo->IsOK()) {
-            keyserializer = NULL; // when joining to 0 rows can't use (LHS) serializer getMinMax will tell slave
-            ActPrintLog(activity, "Suppressing key serializer on master");
+//            keyserializer = NULL; // when joining to 0 rows can't use (LHS) serializer getMinMax will tell slave
```

Comment:
Probably and possibly the whole check/setting of keyserializer=NULL.
It used to cause the slave keyserialize to be muted too,
I wasn't sure what it was originally intended for, but considering the old sort code did not use the keyserializer at all, not much.. 
I should check a case where there 0 rows as a sanity check though, see if anything i haven't spotted falls out.